src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/MvtReader.java

```diff
@@ -71,8 +68,12 @@ public static JtsMvt loadMvt(File file,
                                  RingClassifier ringClassifier) throws IOException {
         final JtsMvt jtsMvt;
 
-        try(final InputStream is = new FileInputStream(file)) {
+        InputStream is = null;
+        try {
```

Comment:
FYI "In addition to the Java 8 language features and APIs above, Android Studio 3.0 and later extends support for try-with-resources to all Android API levels.".

Source: https://developer.android.com/studio/write/java8-support.html

Reply:
Good catch, really don't want to give up try-with-resources.

Reply:
Thanks for finding this. Android Studio 3.0.1 complained that this would not be correct for API level 15, but probably I didn't had the "desugar" setup properly. I will revert this change then.

Reply:
>  but probably I didn't had the "desugar" setup properly

@boldtrn what do you mean?
Android Studio with default settings should work.

Reply:
Yeah I am actually not sure. Android Studio complaint about this, but only after running `Analyze->Inspect Code`. So maybe it was more of warning...